### PR TITLE
Bug 1214322 - Update Django Rest Framework to v3.2

### PR DESCRIPTION
**Pull request by bobsilverberg**
Note that this removes the ability to update users via the API, which was never needed nor used.

@glogiotatidis or @akatsoulas r?

### CODE REVIEW

**Comment by akatsoulas** [comment]
@bobsilverberg sure, I 'll review it no later than tomorrow

**Comment by bobsilverberg** [comment]
Thanks @akatsoulas !

File: oneanddone/tasks/serializers.py [hunk 1]
```diff
@@ -3,12 +3,18 @@
...
+                                     TaskProject, TaskTeam, TaskType)
 
 
 class TaskAttemptSerializer(serializers.ModelSerializer):
 
-    user = serializers.SlugRelatedField(many=False, slug_field='email')
+    user = serializers.SlugRelatedField(
+        many=False,
+        queryset=User.objects.all(),
+        slug_field='email')
```
Comment (akatsoulas) [inline oneanddone/tasks/serializers.py]: Since this seems to be a read-write field, have you consider using `unique=True`?

Comment (bobsilverberg) [inline oneanddone/tasks/serializers.py]: I don't think I understand what `unique=True` would do here. A given `user` could have multiple `TaskAttempt`s, so is `unique=True` accurate?

Comment (akatsoulas) [inline oneanddone/tasks/serializers.py]: Indeed for this serializer you don't need this, ignore the comment above.

Comment (akatsoulas) [inline oneanddone/tasks/serializers.py]: `unique=True` in your model will make sure that you have unique emails. If there are two identical emails in your db, probably the serializer above will produce an error.

**Comment by akatsoulas** [comment]
Code looks good but I haven't tested this locally. r+

**Comment by bobsilverberg** [comment]
Thanks for the review, @akatsoulas. Based on our conversation above and your r+ I think this is ready to merge. I have another one related to the API which I will be issuing shortly.

**Pull request merged**